README.md

```diff
@@ -1,38 +1,32 @@
 # d2l-fetch-auth
-Provides a middleware function for wrapping a window.Request object with d2l authentication for use with d2l-fetch.
 
-## Setup
+[![NPM version](https://img.shields.io/npm/v/d2l-fetch-auth.svg)](https://www.npmjs.org/package/d2l-fetch-auth)
 
-```sh
-npm ci
-```
+Provides a middleware function for wrapping a window.Request object with d2l authentication for use with d2l-fetch.
 
-## Build
+## Installation
 
-```sh
-npm run build
+Install from NPM:
+
+```shell
+npm install d2l-fetch-auth
 ```
 
 ## Usage
 
 Reference the script in your html after your reference to `d2l-fetch` (see [here](https://github.com/Brightspace/d2l-fetch) for details on d2l-fetch):
 
-Install `d2l-fetch-auth` via npm:
-```sh
-npm install Brightspace/d2l-fetch-auth
-```
-
 ```html
 <script type="module">
-import auth from 'd2l-fetch-auth/dist/d2l-fetch-auth.js';
+import auth from 'd2l-fetch-auth/d2l-fetch-auth.js';
```

Comment:
These paths were incorrect and those files didn't actually exist. The ones at the root will include `d2lfetch` if you haven't, and are just cleaner paths.

Reply:
Then do we even need to be making the `dist` version, or is it just for backwards compatibility at this point?

Reply:
This repo is a little special in that it's set up for both ESM and CJS usage. The `dist` directory is CJS, and the `es6` directory is ESM -- both are transpiled using Rollup.

In the root, `index.js` exposes the CJS things from `dist` and `index.es6.js` exposes the ESM things. Because the `package.json` has both `"main": "index.js"` and `"module": "index.es6.js"` that's how it will use one if you `require()` it and another if you `import` it.

Unfortunately, because the docs were wrong, people have been importing from `es6` ([example](https://search.d2l.dev/xref/Brightspace/gbl-map-viewer/src/services/background-service.js?r=f52b6b62#2))and potentially require-ing from `dist` (can't quickly find an example of that). They've even been [importing directly from src!](https://search.d2l.dev/xref/Brightspace/content-components/core/d2l-content-viewer/src/clients/hypermedia-client.js?r=c3da1e8c#3).

I think the real solution here is to make this package ESM-only with a major version bump, remove all the Rollup stuff and then use the `exports` feature in `package.json` to lock down what people can actually import. I'm not sure I have the time for that right now.

Reply:
Makes sense!